server/src/main/java/com/linecorp/centraldogma/server/internal/api/converter/CommitMessageRequestConverter.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2018 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.centraldogma.server.internal.api.converter;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Strings.isNullOrEmpty;
+
+import java.nio.charset.StandardCharsets;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+
+import com.linecorp.armeria.common.AggregatedHttpMessage;
+import com.linecorp.armeria.common.MediaType;
+import com.linecorp.armeria.server.ServiceRequestContext;
+import com.linecorp.armeria.server.annotation.RequestConverterFunction;
+import com.linecorp.centraldogma.internal.Jackson;
+import com.linecorp.centraldogma.internal.api.v1.CommitMessageDto;
+
+/**
+ * A request converter that converts to {@link CommitMessageDto}.
+ */
+public final class CommitMessageRequestConverter implements RequestConverterFunction {
+
+    @Override
+    public Object convertRequest(ServiceRequestContext ctx, AggregatedHttpMessage request,
+                                 Class<?> expectedResultType) throws Exception {
+        if (expectedResultType.isAssignableFrom(CommitMessageDto.class)) {
```

Comment:
- Shouldn't this be reversed? i.e. `CommitMessageDto.class.isAssignableFrom(expectedResultType)`

  ```
  $ jjs
  jjs> java.lang.String.class.isAssignableFrom(java.lang.String.class)
  true
  jjs> java.lang.String.class.isAssignableFrom(java.lang.Object.class)
  false
  jjs> java.lang.Object.class.isAssignableFrom(java.lang.String.class)
  true
  ```
- Please review all `RequestConverterFunction` implementations just in case the same mistakes were made.

Reply:
Just realized that simply reversing `isAssignableFrom()` does not make sense because we won't be able to cast `CommitMessageDto` object to `SubtypeOfCommitMessageDto`.